src/test/unit/rcdevice_unittest.cc.txt

```diff
@@ -29,44 +29,60 @@ extern "C" {
     #include "common/maths.h"
     #include "common/utils.h"
     #include "common/streambuf.h"
-    #include "common/time.h"
-
-    #include "config/parameter_group.h"
-    #include "config/parameter_group_ids.h"
 
     #include "fc/rc_controls.h"
     #include "fc/rc_modes.h"
 
+    #include "drivers/serial.h"
```

Comment:
This should be before the `"fc/...` includes (they're in alphabetical order). Could you please make the tests run too?

Reply:
OK